src/CodeGeneration/ApiGenerator/Generator/CodeGenerator.cs

```diff
@@ -9,16 +9,32 @@ namespace ApiGenerator.Generator
 	//TODO this should be in views and models
 	public static class CodeGenerator
 	{
+		public static string CatFormatPropertyGenerator(string type, string name, string key, string setter) =>
+			  $"public {type} {name} {{ "
+			+ $"	get => Q<{type}>(\"{key}\");"
+			+ $"	set {{ "
+			+ $"		Q(\"{key}\", {setter});"
+			+ $"		if (RequestConfiguration == null) "
+			+ $"			RequestConfiguration = new RequestConfiguration();"
+			+ $""
+			+ $"		RequestConfiguration.Accept = AcceptHeaderFromFormat({setter});"
```

Comment:
Can we hide this behind a single lined method?

Reply:
I've moved this into its own method